src/themes/DarkTheme.js

```diff
@@ -32,6 +32,31 @@ export default {
     backgroundColor: colors.shark,
   },
 
+  inputWindow: {
```

Comment:
Please, keep in themes files only variables that we use to style components.
If we want to use some of those colors to style the story decorator, use those colors directly from the `colors` object.